RealmSwift/Decimal128.swift

```diff
@@ -250,8 +250,8 @@ extension Decimal128 {
     ///
     /// - Parameter other: The Decimal128 value to calculate the distance to.
     /// - Returns: The distance from this value to `other`.
-    public func distance(to other: Decimal128) -> Stride {
-        other - self
+    public func distance(to other: Decimal128) -> Decimal128 {
+        unsafeDowncast(other.decimalNumber(bySubtracting: self), to: Decimal128.self)
```

Comment:
Overloaded operators take a really long time to type-check so each of these were taking close to 100ms to build and now are <1ms. `unsafeDowncast()` ended up not speeding up the build but made the release binary a bit smaller (but not enough to justify listing it in the changelog).

Reply:
Is it worth adding some comments to say why you use `unsafeDowncast`? Its not initially apparent that it improves build times.

Reply:
But I suppose you could just always trace the git blame